src/mobile-token/hooks/useGetSignedTokenQuery.tsx

```diff
@@ -22,7 +22,7 @@ export const useGetSignedTokenQuery = () => {
       try {
         return await mobileTokenClient.encode(nativeToken);
       } catch (err: any) {
-        notifyBugsnag(err, {
+        notifyBugsnag(err, `GetSignedTokenError - ${err}`, {
```

Comment:
I am just asking a bit here, since this parameter is a hash should it not include ${err} since it can make the hash to change and not group against GetSignedTokenError?

Reply:
That is what I asked on the `Review Notes` above, 

> Please let me know what do you think about the grouping, particularly errors that might need to be grouped together, but may have different error specification, such as the GetSignedTokenError, should we put the error type on the group (to differentiate them), or should we put the error type on the metadata instead (to group all of them together).

It is open for discussion and I think `${err}` should be in metadata instead, but I'm up for suggestions 😄

Reply:
I wrote a comment in the task :)